### ISSUE TITLE
CHD Hashing

### ISSUE THREAD

**Issue by ZeROOFALL** [notaz/pcsx_rearmed#345]
Can CHD Hash checking be enabled to allow retroachievements to work properly? I've converted my games from BIN/CUE to CHD and they work flawlessly but the only issue I have is that I can not get retroachievements to work with them after the conversion because PCSX_ReARMed does not check the hash's

**Comment by notaz** [comment]
It this really a job for the emulation core? Whatever frontend you're using could do it. Or do you know any other libretro core that does this? I don't see anything hashing related in the libretro API.

**Comment by ZeROOFALL** [comment]
I know some cores are capable of it. I did some digging (way above my head) but I might have found a answer. 

"For the longest time, CHD games were not working with RetroAchievements on the UWP builds of RetroArch. This was due to the frontend not being built with both CHD and 7zip support, which are required for CHD files to be hashed correctly, and subsequently work with RetroAchievements.

The solution to this is simple, simply add the HAVE_CHD, HAVE_7ZIP, and _7ZIP_ST build flags to the UWP builds. This effectively solves the issue, while adding 7zip support to the frontend as a nice bonus too."

https://github.com/libretro/RetroArch/pull/13669

I'm using the latest switch retroarch build and can see under system that 7zip is supported, and assuming CHD is because I can launch that format... I guess my other question would be if not here, where should I forward this info to?
